src/lib/components/toggle/Toggle.component.js

```diff
@@ -19,77 +19,81 @@ const Switch = styled.label`
   position: relative;
   display: inline-block;
   width: 24px;
-  height: 24px;
-  margin-right: ${defaultTheme.padding.small};
+  height: 14px;
 
-  input {
+  & input {
     opacity: 0;
     width: 0;
     height: 0;
   }
+`;
 
-  ${(props) => {
-    const { secondary } = getTheme(props);
-    return css`
-      .sc-slider {
-        position: absolute;
-        cursor: pointer;
-        top: 10px;
-        left: 0;
-        right: 0;
-        background-color: ${defaultTheme.grayLight};
-        transition: 0.5s;
-        height: 4px;
-        border-radius: 4px;
-
-        &:hover {
-          &:before {
-            box-shadow: 0 0 3px ${defaultTheme.grayLight};
-          }
-        }
-      }
+const Slider = styled.span`
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: ${(props) => getTheme(props).backgroundLevel1};
+  -webkit-transition: 0.4s;
```

Comment:
Styled component automatically add vendor prefixing to css rules (https://styled-components.com/docs/basics#motivation) so this line can be removed